nshimiyejayd/my-brand
# Implementing my brand section one

#### What does this PR do?
This PR implements my brand

#### Description of Task to be completed?
- Add Home page
- Add About Page
- Add Skills page
- Add Contact page
- Add Blogs page
- Add Login

#### How should this be manually tested?
- Clone the[repo](https://github.com/nshimiyejayd/my-brand)
-  Checkout branch `feat-section-1`
- Open the index.html in your browser to check all pages

#### What are the relevant pivotal tracker stories?
[#TmnFdrKg](https://trello.com/b/TmnFdrKg/atlp-c-25-my-brand)
